resources/language/resource.language.zh_cn/strings.po

```diff
@@ -0,0 +1,1027 @@
+# Kodi Media Center language file
+# Addon Name: Netflix
+# Addon id: plugin.video.netflix
+# Addon Provider: libdev + jojo + asciidisco + caphm + CastagnaIT
+msgid ""
+msgstr ""
+"Project-Id-Version: plugin.video.netflix\n"
+"Report-Msgid-Bugs-To: https://github.com/CastagnaIT/plugin.video.netflix\n"
+"POT-Creation-Date: 2017-07-24 16:15+0000\n"
+"PO-Revision-Date: 2020-07-09 16:40+0000\n"
+"Last-Translator: shaolin-kongfu <>\n"
+"Language-Team: Chinese\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: zh_CN\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgctxt "Addon Summary"
+msgid "Netflix"
+msgstr "Netflix"
+
+msgctxt "Addon Description"
+msgid "Netflix VOD Services Add-on"
+msgstr "Netflix VOD服务附加组件"
+
+msgctxt "Addon Disclaimer"
+msgid "Some parts of this add-on may not be legal in your country of residence - please check with your local laws before installing."
+msgstr "此加载项的某些部分在您居住的国家可能不合法-请在安装前与您当地的法律联系。"
+
+msgctxt "#30001"
+msgid "Recommendations"
+msgstr "建议"
+
+msgctxt "#30002"
+msgid "Enter PIN to watch restricted content"
+msgstr "输入PIN来观看受限内容"
+
+# Unused 30003
+
+msgctxt "#30004"
+msgid "Password"
+msgstr "密码"
+
+msgctxt "#30005"
+msgid "E-mail"
+msgstr "邮箱"
+
+msgctxt "#30006"
+msgid "Enter PIN to access this profile"
+msgstr "输入PIN即可访问此配置文件"
+
+msgctxt "#30007"
+msgid "Parental Control PIN (4 digits)"
+msgstr "家长控制PIN（4位数字）"
+
+msgctxt "#30008"
+msgid "Login failed"
+msgstr "登录失败"
+
+msgctxt "#30009"
+msgid "Please check your e-mail and password"
+msgstr "请检查您的邮箱和密码"
+
+msgctxt "#30010"
+msgid "Lists of all kinds"
+msgstr "所有种类的列表"
+
+# Unused 30011, 30012, 30013
+
+msgctxt "#30014"
+msgid "Account"
+msgstr "账户"
+
+msgctxt "#30015"
+msgid "Other options"
+msgstr "其他选项"
+
+msgctxt "#30016"
+msgid "ESN"
+msgstr "ESN"
+
+msgctxt "#30017"
+msgid "Logout"
+msgstr "注销"
+
+msgctxt "#30018"
+msgid "Export to library"
+msgstr "导出到库"
+
+msgctxt "#30019"
+msgid "Rate on Netflix"
+msgstr "在Netflix上评分"
+
+msgctxt "#30020"
+msgid "Remove from 'My list'"
+msgstr "从'我的列表'中删除"
+
+msgctxt "#30021"
+msgid "Add to 'My list'"
+msgstr "添加到'我的列表'"
+
+msgctxt "#30022"
+msgid "(between 0 & 10)"
+msgstr "（介于0和10之间）"
+
+msgctxt "#30023"
+msgid "Expert"
+msgstr "专家"
+
+msgctxt "#30024"
+msgid "SSL verification"
+msgstr "SSL验证"
+
+msgctxt "#30025"
+msgid "Library"
+msgstr "库"
+
+msgctxt "#30026"
+msgid "Enable custom library folder"
+msgstr "启用自定义库文件夹"
+
+msgctxt "#30027"
+msgid "Custom library path"
+msgstr "自定义库路径"
+
+msgctxt "#30028"
+msgid "Playback error"
+msgstr "播放错误"
+
+# Unused 30029
+
+msgctxt "#30030"
+msgid "Remove from library"
+msgstr "从库中删除"
+
+msgctxt "#30031"
+msgid "General"
+msgstr "常规"
+
+# Unused 30032
+
+msgctxt "#30033"
+msgid "Enable Dolby Digital Plus (DDPlus HQ and Atmos on Premium)"
+msgstr "启用杜比 Plus"
```

Comment:
Here is needed to specify also: DDPlus HQ and Atmos on Premium

Reply:
OK,I have fixed it.

Reply:
to be safe i just want to try explain better
DDPlus HQ and Atmos sound type are both enabled only with "Premium netflix account"
is written only "premium" to be concise

Reply:
I just get your point now.
How about translate this sentence like this:"Enable DDPlus HQ and Atmos for Premium accounts"
Is it ok?